frontend/src/store/authSlice.js

```diff
@@ -0,0 +1,136 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+
+const TOKEN = "token";
+/**
+ * async thunk that authorizes a user through an AJAX request
+ * receives the token from local storage when user is logged in
+ * @returns authentication confirmation
+ * @catches error if database request goes wrong
+ */
+export const me = createAsyncThunk("auth/me", async (thunkAPI) => {
+  const token = window.localStorage.getItem(TOKEN);
+  try {
+    if (token) {
+      const { data } = await axios.get("http://localhost:5000/auth/me", {
```

Comment:
Start refactoring these `axios` requests so they don't point to `localhost` anymore - you can put the url in a `.env` that toggles based on whether it's in dev or production